utility/mirrormanager2.spec

```diff
@@ -208,7 +206,7 @@ Base directories used by multiple subpackages
 
 
 %prep
-%setup -q
+%setup -q -n mirrormanager2-0.17
```

Comment:
This also seems unnecessary.

Reply:
True, the name is the same, removing. I wonder what the `-q` option means,  I don't see it anywhere in the documentation: http://ftp.rpm.org/max-rpm/s1-rpm-inside-macros.html

Reply:
I think `-q` is to not print out the output of untarring.